csharp/extractor/Semmle.Extraction.CSharp.Standalone/Runtime.cs

```diff
@@ -63,7 +63,20 @@ public static IEnumerable<string> DesktopRuntimes
             }
         }
 
-        public static IEnumerable<string> Runtimes
+        /// <summary>
+        /// Gets the .NET runtime location to use for extraction
+        /// </summary>
+        public static string GetRuntime(bool useSelfContained)
+        {
+            if (useSelfContained)
+            {
+                return ExecutingRuntime;
+            }
+
+            return Runtimes.First();
```

Comment:
Perhaps ` => useSelfContained ? ExecutingRuntime : Runtimes.First()`.

Reply:
Pushed an extra commit to fix this.